app/src/main/java/org/oppia/android/app/player/state/SelectionInteractionView.kt

```diff
@@ -108,7 +113,7 @@ class SelectionInteractionView @JvmOverloads constructor(
           .build()
       SelectionItemInputType.RADIO_BUTTONS ->
         BindableAdapter.SingleTypeBuilder
-          .newBuilder<SelectionInteractionContentViewModel>()
+          .Factory(fragment).create<SelectionInteractionContentViewModel>()
```

Comment:
We should use the injected factory here rather than reconstructing it.

Reply:
Applied changes.